integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorWlsUpgrade.java

```diff
@@ -179,6 +203,75 @@ void testOperatorWlsUpgradeFrom338ToLatest(String domainType) {
+  @Disabled()
+  @Test
+  @DisplayName("Upgrade 3.3.8 Domain(v8 schema) with Auxiliary Image to latest")
+  void testOperatorWlsAuxDomainUpgradeFrom338ToLatest() {
+    logger.info("Starting test to upgrade Domain with Auxiliary Image with v8 schema to latest ");
+
+    installOldOperator("3.3.8");
+    createSecrets();
+   
+    final String auxiliaryImagePath = "/auxiliary";
+    List<String> archiveList = Collections.singletonList(ARCHIVE_DIR + "/" + MII_BASIC_APP_NAME + ".zip");
+    List<String> modelList = new ArrayList<>();
+    modelList.add(MODEL_DIR + "/" + MII_BASIC_WDT_MODEL_FILE);
+    modelList.add(MODEL_DIR + "/model.jms2.yaml");
+    logger.info("creating auxiliary image {0}:{1} using imagetool.sh ", miiAuxiliaryImage, MII_BASIC_IMAGE_TAG);
+    createPushAuxiliaryImageWithDomainConfig(miiAuxiliaryImage, archiveList, modelList);
+
+    String auxImage = miiAuxiliaryImage + ":" + MII_BASIC_IMAGE_TAG;
+    Map<String, String> templateMap  = new HashMap();
+    templateMap.put("DOMAIN_NS", domainNamespace);
+    templateMap.put("DOMAIN_UID", domainUid);
+    templateMap.put("AUX_IMAGE", auxImage);
+    templateMap.put("BASE_IMAGE", WEBLOGIC_IMAGE_TO_USE_IN_SPEC);
+    templateMap.put("API_VERSION", "v8");
+    Path srcDomainFile = Paths.get(RESOURCE_DIR,
+        "upgrade", "auxiliary.domain.template.v8.yaml");
+    Path targetDomainFile = assertDoesNotThrow(
+        () -> generateFileFromTemplate(srcDomainFile.toString(),
+        "domain.yaml", templateMap));
+    logger.info("Generated Domain Resource file {0}", targetDomainFile);
+
+    // run kubectl to create the domain
+    logger.info("Run kubectl to create the domain");
+    CommandParams params = new CommandParams().defaults();
+    params.command("kubectl apply -f "
+            + Paths.get(WORK_DIR + "/domain.yaml").toString());
+    boolean result = Command.withParams(params).execute();
+    assertTrue(result, "Failed to create domain custom resource");
+
+    // wait for the domain to exist
+    logger.info("Checking for domain custom resource in namespace {0}", domainNamespace);
+    testUntil(
+        domainExists(domainUid, "v8", domainNamespace),
+        logger,
+        "domain {0} to be created in namespace {1}",
+        domainUid,
+        domainNamespace);
+    checkDomainStarted(domainUid, domainNamespace);
+    LinkedHashMap<String, OffsetDateTime> pods = new LinkedHashMap<>();
+    pods.put(adminServerPodName, getPodCreationTime(domainNamespace, adminServerPodName));
+    // get the creation time of the managed server pods before upgrading the operator
+    for (int i = 1; i <= replicaCount; i++) {
+      pods.put(managedServerPodNamePrefix + i, getPodCreationTime(domainNamespace, managedServerPodNamePrefix + i));
+    }
+
+    // verify there is no status condition type Completed before upgrading to Latest
+    verifyDomainStatusConditionTypeDoesNotExist(domainUid, domainNamespace,
```

Comment:
do we need to check domain status after upgrade?

Reply:
I just took the same code from the existing test method.

Reply:
Removed the redundant check